miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -25,6 +25,12 @@ const.INPUT_VAULT_ROOT_PTR=3
 # The memory address at which the output vault root is stored
 const.OUTPUT_VAULT_ROOT_PTR=4
 
+# The memory address at which the pointer to the data of the currently accessing account is stored
+const.CURRENT_ACCOUNT_DATA_OFFSET=5
```

Comment:
Should we rename this to `CURRENT_ACCOUNT_DATA_PTR` since this stores a memory address rather than a relative offset?